rest/route/commit_queue.go

```diff
@@ -124,3 +125,46 @@ func (cq *commitQueueClearAllHandler) Run(ctx context.Context) gimlet.Responder
 		ClearedCount int `json:"cleared_count"`
 	}{clearedCount})
 }
+
+type commitQueueEnqueueItemHandler struct {
+	item string
+
+	sc data.Connector
+}
+
+func makeCommitQueueEnqueueItem(sc data.Connector) gimlet.RouteHandler {
+	return &commitQueueEnqueueItemHandler{
+		sc: sc,
+	}
+}
+
+func (cq commitQueueEnqueueItemHandler) Factory() gimlet.RouteHandler {
+	return &commitQueueEnqueueItemHandler{
+		sc: cq.sc,
+	}
+}
+
+func (cq *commitQueueEnqueueItemHandler) Parse(ctx context.Context, r *http.Request) error {
+	vars := gimlet.GetVars(r)
+	cq.item = vars["item"]
+
+	return nil
+}
+
+func (cq *commitQueueEnqueueItemHandler) Run(ctx context.Context) gimlet.Responder {
+	patch, err := cq.sc.FindPatchById(cq.item)
```

Comment:
I might push the find patch into the Parse part, which would make it easier to test/find 404 errors...

Reply:
done.